### DISCUSSION TITLE
Netdata Alerts Configuration Manager is now live! ⏰🚀

### DISCUSSION THREAD

**Discussion by hugovalente-pm** [netdata/netdata#16657] [Announcements]
<img width="713" alt="alert-config-manager-64e543c623032a3c874d06933d080fb4" src="https://github.com/netdata/netdata/assets/82235632/4c39cc8e-854a-4f15-b5fa-8e08947a2ed5">

**The wait is finally over!** 

Based on requests from a number of users on the Netdata Community, we present to you the Alerts Configuration Manager which provides you a wizard to create and manage alerts for your infrastructure monitoring, directly from the **Netdata Dashboards**. 

Learn more about the feature on 
* [Youtube Video](https://www.youtube.com/watch?v=abCrs-rn97U&t=304s) from the meetup
* [Documentation](https://learn.netdata.cloud/docs/alerting/creating-alerts-with-the-alerts-configuration-manager)
* Getting started [Blog post ](https://blog.netdata.cloud/netdata-alerts-configuration-manager/)

PS: This feature is only available for our Business subscriptions.

**Comment by jasalt** [comment]
Thank's for the good open documentation on this project, it is interesting to see how these things are being built behind the scenes.
